store/src/java/com/zimbra/cs/account/soap/SoapDistributionList.java

```diff
@@ -81,11 +81,20 @@
                 SoapProvisioning.getAttrs(e), prov);
         addDlm(e, getRawAttrs());
     }
-
+    
+    //ZBUG-651: This bug is about to get correct member details for gadl command;
```

Comment:
this comment `//ZBUG-651: This bug is about to get correct member details for gadl command;` is not required here can move it to method body where the change is made

Reply:
Done.